BREAKING.md

```diff
@@ -61,3 +61,8 @@ When preparing a release that can include breaking changes, consider applying ch
     - Owner: @timoffex
     - Deprecated after 0.19.8 (https://github.com/wandb/wandb/pull/9557)
     - Can do after September 2025
+
+- Deprecate `data_is_not_path` and add `path` keyword argument to explicitly handle files.
```

Comment:
This doesn't give enough context. It should mention that this is referring to the `wandb.Html` constructor.